### ISSUE TITLE
Add a way to bypass form validation when calling an @Action on the server.

### ISSUE THREAD

**Issue by gilberto-torrezan** [jefalbino/jsmart-web#2]
I have the following scenario: a form with an input text and a select. The options on the select are dependent on what is typed at the input box. (It's not a case of an autocomplete - both inputs, from the input and from the select, are separate data).

So when the user finishes typing the text on the input text, I have to query the server for the possible values (related to what was typed) and show them as options inside the select.

Both the input text and the select are mandatory fields: the user must type and must select something before submitting the form.

The problem is: this action to fill the options of the select is not the final submit. The request is never made if the validator on any of the fields blocks the request.

I need a way to tell the framework that this Action does not need to be validated - the select can be empty before asking for the options.

Sample code of the action:

``` java
@Action(forIds = "my-input-text", event = "blur", onComplete = "updateMySelectWithTheOptions")
public void updateSelectOptions(@Arg(bindTo = "my-input-text") String text) {        
    //do something with the text
    //return the options to the select
}
```

**Comment by jefalbino** [comment]
Investigating your scenario I can say that once your input element is inside a form and your event binding via @Action uses the Ajax, the default behavior is to carry form values along with the request and also validate elements that contains validate component embedded on it.
In order to provide more functionalities we can add a new feature to skipValidation when request is done, so this attribute will be available as @Action attribute and also on ajax tag.
By default the validation will continues to happen for backward compatibility, but in your annotation you can use this new attribute, for instance:

``` java
@Action(forIds = "my-input-text", event = "blur", onComplete = "updateMySelectWithTheOptions",
        skipValidation=true)
public void updateSelectOptions(@Arg(bindTo = "my-input-text") String text) {        
    //do something with the text
    //return the options to the select
}
```

Just keep in mind that the validation will be skipped, and it will be your responsibility to handle it on server via @PreAction or via @Action.beforeSend attribute which you can specify the JavaScript function to be called before Ajax is performed.

Please let me know if this new feature may help you ;)

**Comment by jefalbino** [comment]
Included skipValidation as attribute for @Action. It is available on 2.1.1-SNAPSHOT. Please let me know if it solves the problem
